### ISSUE TITLE
External Playlist use results only in 403 on /ext/stream location

### ISSUE THREAD

**Issue by DutchComputerKid** [tesshucom/jpsonic#1969]
## Problem description

While using the (legacy) feature of external playlists for problem-free DSD and ASIO playback, any file played using the external player feature is given a 403 by the server.
I have tried to change the link to just /stream? instead of /ext/stream? but the output is the login page.

```
Status
    403
Error
    Forbidden
Message
Path
    /ext/stream
```
### Steps to reproduce

1. *Turn on external playlists in legacy features*
2. *Set the player to use external player/playlists, both will result the same*
3. *Try to play in software such as Foobar or try opening the link in a browser.*

## System information

 * **Jpsonic version**: Docker: Jpsonic 111.5.11 – 2022-12-08, build b43f422d838a50bc21c62b2e21a1080edb5c058d
 * **Operating system**: Debian 11 amd64
 * **Java version**: openjdk 17.0.4 2022-07-19
 * **Proxy server**:  None
 * **Client**: Firefox & Edge
 * **Language**: English
 * **Tomcat/standalone**: Docker

**Comment by tesshucom** [comment]
I use UPnP when checking with foobar, but I was able to confirm "Network error" even though I didn't change the settings. something might be happening, Browsing with UPNP is possible, but as you pointed out, "Network error" seems to occur when connecting via /ext. For curl or browser, "Timeout". This is serious, so if we investigate and find the problem, there is a high possibility that a fixed version will be released.

(on the other hand ... no ideas for #1968 so far.)

**Comment by DutchComputerKid** [comment]
Subsonic never really worked API-wise with files like DSD when using MusicBee, so for a couple years now external playlists have worked flawlessly instead.
In any case, good to see all of this! If anything pops up to help debugging, I'd love to help

**Comment by tesshucom** [comment]
Thank you. Haven't seen it yet, but I think it can probably be fixed. Before the trouble occurred, normal operation was confirmed with the local renderer of BubbleUPnP. DSD is a slightly strange format, but I think there is a possibility that it can be played with foobar2000. Please wait a little bit ...

**Comment by tesshucom** [comment]
UPnP doesn't seem to have any issues. It was a feature that had a lot of degradations, so I thought something else had happened. It was just a mistake in the UPnP URL setting .... 

When I checked with foobar2000, I can confirm that streaming is possible.

![image](https://user-images.githubusercontent.com/27724847/209445168-928af06d-d8e9-4cc2-9bee-5ca709c055ed.png)

foobar2000 may be relatively easy to check. Here's what it looks like on the taskbar: 

![image](https://user-images.githubusercontent.com/27724847/209445179-8745ad75-957a-4f72-a453-3c4056a9398a.png)

However, when I tried to access from the m3u URL, I was able to confirm the phenomenon you pointed out. 

![image](https://user-images.githubusercontent.com/27724847/209445323-0dc3482e-f42f-4b40-a4df-2bdd5db3f513.png)

It may be a problem with the URL generation. Probably not so serious.

**Comment by DutchComputerKid** [comment]
If you use PCM conversion, maybe.

But in any case, UPnP shows me:
```
Unable to open item for playback (Object not found):
"http://192.168.2.182/ext/stream?id=5455&player=4&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJwYXRoIjoiL2V4dC9zdHJlYW0_aWQ9NTQ1NSZwbGF5ZXI9NCIsImV4cCI6MTY3MjUyNjgzOX0.4dKZdUSjRL8_s_d7BOaV3YOUQ4dXePUSrRWWmt11AWE.flac"
```
or
```
Unable to open item for playback (Object not found):
"http://192.168.2.182/ext/stream?id=49543&player=4&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJwYXRoIjoiL2V4dC9zdHJlYW0_aWQ9NDk1NDMmcGxheWVyPTQiLCJleHAiOjE2NzI1MjY4MjB9.N1_g3b9HqrRgi7hPPaRuGtL13WE4NqaK3yLXDDaSiBw.dsf"
```

No errors being logged either to give any sense of where in the chain this is happening.

**Comment by tesshucom** [comment]
It's worth to try adding Port designation...

![image](https://user-images.githubusercontent.com/27724847/209460469-caeaa4d5-b0eb-475d-9781-663579024ee3.png)

If there is no problem with network settings such as firewall, I think it will work.

**Comment by tesshucom** [comment]
Fixed version [v111.5.13](https://github.com/tesshucom/jpsonic/actions/runs/3776416793) released. It will improve the cooperation with foobar2000 by m3u.

Plese use "External player with playlist".

![image](https://user-images.githubusercontent.com/27724847/209475370-62262dc3-57ac-4171-982d-fe651abf3b1f.png)

We are considering deleting "External player" in the future. (no longer maintained) 
"**External player with playlist**" will be kept.

I'm not sure many users understand the exact difference between them... 

 - "External player" issues a static URL and dynamically responds with the "currently playing song" on the server side. 
   - Twenty years ago, the function might have had some significance. With this function, it was possible to "operate the play queue in the browser and let foobar2000 or Real Player play". For security reasons, most browsers do not allow m3u to operate external programs nowadays. So it's already a rotten feature.
   - It's an anti-pattern to randomly reply different resources to a unique URL. Not well designed. In fact, it adds unnecessary complexity to many features. It is also outdated in this regard.
 - "**External player with playlist**" assumes that the full list of songs by m3u is loaded into foobar2000 etc. and controlled by an application such as foobar2000. As long as it works correctly, I think it will probably meet the needs of users. Most people using m3u today have this use case in mind.
